src/platform/packages/shared/kbn-lock-manager/src/setup_lock_manager_index.ts

```diff
@@ -35,7 +35,16 @@ export async function removeLockIndexWithIncorrectMappings(
     return;
   }
 
-  const { mappings } = res[LOCKS_CONCRETE_INDEX_NAME];
+  const indexNames = Object.keys(res);
+  if (indexNames.length === 0) {
+    logger.debug(`No mappings found for "${LOCKS_CONCRETE_INDEX_NAME}"`);
```

Comment:
This should probably be a warning. We already check for `isNotFoundError` in the catch handler so theoretically the response should return a list of mappings - and if it cannot it should be flagged.

```suggestion
    logger.warn(`No mappings found for "${LOCKS_CONCRETE_INDEX_NAME}"`);
```

Reply:
Ah right, so very unlikely to happen and should be a warning - makes sense to me.